### DISCUSSION TITLE
Issues logging into File Manager

### DISCUSSION THREAD

**Discussion by garamoniker** [GeekFunkLabs/fluidpatcher#46] [Q&A] [answered]
Hey folks, excuse my new user-tier question.

It seems the Username and Password in the Squishbox Manual for the Squishbox File Manager are no longer valid. Can I get the new ones or am I doing something wrong, perhaps?

Thanks!
![Squishbox Manager](https://user-images.githubusercontent.com/108605251/177028161-21f18118-e276-425e-a0f8-b444fa5c95fb.jpg)

**Comment by albedozero** [comment, marked as answer]
No problem - to reset the file manager password, run the install/update script as normal by entering `curl -L git.io/squishbox | bash` and reply _"No"_ or _"No change"_ to all the questions except _"Set up web-based file manager?"_. It will then prompt you for a new username and password.

If you're into it, you can instead manually edit the file _/var/www/html/index.php_. The [tinyfilemanager docs](https://github.com/prasathmani/tinyfilemanager/wiki/Security-and-User-Management) explain how to add users and hashed passwords.

**Reply by garamoniker** [reply]
Thanks so much.
